app/contacts/contact_websocket.py

```diff
@@ -1,43 +1,119 @@
 import asyncio
+import json
 import websockets
+import inspect
 
-from app.utility.base_world import BaseWorld
+from datetime import datetime, timezone
 
+from app.service.interfaces.i_event_svc import EventServiceInterface
+from app.utility.base_service import BaseService
 
-class Contact(BaseWorld):
 
-    def __init__(self, services):
-        self.name = 'websocket'
-        self.description = 'Accept data through web sockets'
-        self.log = self.create_logger('contact_websocket')
-        self.log.level = 100
-        self.handler = Handler(services)
-        self.stop_future = asyncio.Future()
+class EventService(EventServiceInterface, BaseService):
 
-    async def start(self):
-        web_socket = self.get_config('app.contact.websocket')
+    def __init__(self):
+        self.log = self.add_service('event_svc', self)
+        self.contact_svc = self.get_service('contact_svc')
+
+        # Ensure WebSocket URI is properly configured
         try:
-            async with websockets.serve(self.handler.handle, *web_socket.split(':'), logger=self.log):
-                await self.stop_future
+            ws_config = self.get_config('app.contact.websocket')
+            if not ws_config:
+                raise ValueError("WebSocket URI is not properly configured.")
+            self.ws_uri = f'ws://{ws_config}'
+        except Exception as e:
+            self.log.error(f"Error setting WebSocket URI: {e}", exc_info=True)
+            self.ws_uri = 'ws://localhost:8888'  # Fallback URI
```

Comment:
```suggestion
            self.ws_uri = 'ws://localhost:7012'  # Fallback URI
```
Port 8888 is typically the default for the server's HTTP traffic, so we can use 7012, which is the default websocket port in the default config file.